demos/SpatialCoop2017/source/SimplePDWorld.hpp

```diff
@@ -111,16 +111,17 @@ class SimplePDWorld {
 
   void Reset() { Setup(r,u,N,E); }
 
-  void Run(size_t steps=-1) {
-    if (steps > E) steps = E;
-    // Run the organisms!
-    size_t end_epoch = epoch + steps;
-    while (epoch < end_epoch) {
+  void Run(size_t steps=1) {
```

Comment:
outside the scope of this pr but omg these single letter variable names

Reply:
Hah, yeah, agreed!